### PR TITLE
Localized file check-in by OneLocBuild Task: Build definition ID 981: Build ID 1582980

### PR DESCRIPTION
**Pull request by MiYanni**
This is the pull request automatically created by the OneLocBuild task in the build process to check-in localized files generated based upon translation source files (.lcl files) handed-back from the downstream localization pipeline. If there are issues in translations, visit https://aka.ms/ceLocBug and log bugs for fixes. The OneLocBuild wiki is https://aka.ms/onelocbuild and the localization process in general is documented at https://aka.ms/AllAboutLoc.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7875)

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/xlf/AssemblyReference.xaml.zh-Hant.xlf [hunk 1]
```diff
@@ -1,97 +1,97 @@
...
+        <note />
+      </trans-unit>
+      <trans-unit id="StringListProperty|Aliases|Description">
+        <source>A comma-delimited list of aliases to this reference.</source>
+        <target state="translated">這個參考的逗號分隔別名清單。</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="BoolProperty|Private|DisplayName">
+        <source>Copy Local</source>
+        <target state="new">Copy Local</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/xlf/AssemblyReference.xaml.zh-Hant.xlf:37]: This looks wrong. If you disable whitespace changes in the diff you can see that this PR just undoes a bunch of translations. It's unclear why it's doing that.

**Review by drewnoakes — COMMENTED** [review]
Not sure about this one. @MiYanni does this seem right to you?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/xlf/AssemblyReference.xaml.zh-Hant.xlf (hunk 1 above)
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/xlf/AssemblyReference.xaml.zh-Hant.xlf:37]: image

**Comment by drewnoakes** [comment]
Actually, I think this follows from #7864 in which I tried to preserve the translations here. Seems like the upstream tooling didn't optimize the change I made, likely having its own source of truth for this data.

**Comment by MiYanni** [comment]
@drewnoakes Yes. I tried the same thing in the past too. You can't force translations to persist if the items themselves change or if you manually change the translations. Since their 'source of truth' doesn't match what we give them, they assume it is invalid and mark it as `new`. As it stands, we shouldn't need to review these PRs. I did have them add that description to the PRs that are generated. It used to just say `null` in the past, such as this: https://github.com/dotnet/project-system/pull/7527#issue-981622816

I'm also working on getting my creds to no longer create these PRs. I figured out how they do it themselves in Arcade. Asking people to see if I can get more info on it.

**Pull request merged**